packages/gantt/frontend/sirius-components-gantt/src/representation/Gantt.tsx

```diff
@@ -285,11 +282,14 @@ export const Gantt = ({
         checkIsHoliday={checkIsHoliday}
         dateMoveStep={dateRounding}
         onWheel={onwheel}
-        ContextualPalette={getContextalPalette({
+        ContextualPalette={getTaskContextalPalette({
           onCreateTask,
-          onDeleteTask: handleDeleteTaskOnContextualPalette,
+          onDeleteTask: (task: Task) => onDeleteTask([task]),
           onEditTask,
         })}
+        TaskDependencyContextualPalette={getTaskDependencyContextalPalette({
```

Comment:
Why these two props start with a capital letter?

Reply:
because it is react components